### DISCUSSION TITLE
Mengatur tampilan di html

### DISCUSSION THREAD

**Discussion by 19807311** [teamalgoritma/community#453] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/114885734/202167077-7ae654d3-e86f-4173-9cea-4daa33f54b14.png)

Halo saya mau tanya, saya screen shot satu project yang saya lihat di github. Bagaimana cara membuat tampilan seperti ini? Beberapa plot/grafik bisa diatur penempatannya / dijejerkan begini sehingga tidak perlu satu-satu ke bawah.
Terima kasih...

**Comment by divakartika** [comment, marked as answer]
Dear @19807311 

Untuk menyusun banyak plot dalam satu halaman, dapat digunakan fungsi `plot_grid()` dari library `cowplot` atau `grid.arrange()` dari library `gridExtra`. Berikut referensi untuk keduanya:

- `plot_grid()`: https://wilkelab.org/cowplot/reference/plot_grid.html
- `grid_arrange()`: https://r-graph-gallery.com/261-multiple-graphs-on-same-page.html

Silakan dicoba, semoga informasi ini membantu.

**Reply by 19807311** [reply]
Baik akan saya coba. Terima kasih Mbak
